### DISCUSSION TITLE
Cannot add Gitea repository to ArgoCD

### DISCUSSION THREAD

**Discussion by orojastech** [argoproj/argo-cd#13929] [General]
Hi, I'm trying to add my Gitea repository (git@gitea-http:3000/myuser/test.git) to ArgoCD (gitea and argocd are in kubernetes in the same argocd namespace) but can't connect, I'm doing it over ssh, I already configured the ssh deploy in Gitea (.pub) and in ArgoCD I configured through SSH adding the private key (-----BEGIN OPENSSH PRIVATE KEY-----), I tried with public and private repositories, also without ports (git @gitea-http/myuser/test.git) as well as trying to configure over HTTPS but since I have my gitea repository with HTTP not working, any help would be appreciated. Thank you.

**Comment by igelkotten** [comment]
In my gitea implementation I have another service for ssh, gitea-ssh(I think you need to use that since you are using ssh git with ssh)

Aside from that are they in the same namespace since you are using gitea without the FQDN of gitea-http?

**Comment by cloudscape-germany** [comment]
Looks like you are using an external port? How did you solve it anyway? @orojastech
